src/detect-fast-pattern.c

```diff
@@ -272,6 +272,11 @@ static int DetectFastPatternSetup(DetectEngineCtx *de_ctx, Signature *s, const c
                 }
             }
         }
+        if ((cd->flags & DETECT_CONTENT_BASE64_DATA) == DETECT_CONTENT_BASE64_DATA) {
```

Comment:
I think this can be replaced by a check like `if (SigMatchListSMBelongsTo(s, pm) == DETECT_SM_LIST_BASE64_DATA)`

Reply:
oh ok. Thank you!